apps/system-e2e/src/tests/islandis/application-system/acceptance/parental-leave.spec.ts

```diff
@@ -16,13 +20,51 @@ import {
   EmailAccount,
   makeEmailAccount,
 } from '../../../../support/email-account'
-import { helpers, locatorByRole } from '../../../../support/locator-helpers'
+import { helpers } from '../../../../support/locator-helpers'
 import { session } from '../../../../support/session'
-import { mockApi } from '../../../../support/api-tools'
 import { setupXroadMocks } from './setup-xroad.mocks'
 
 test.use({ baseURL: urls.islandisBaseUrl })
 
+async function getEmployerEmailAndApprove(employer: EmailAccount, page: Page) {
+  const { proceed } = helpers(page)
+
+  const email = await employer.getLastEmail(6)
+
+  if (email && typeof email.html === 'string') {
+    const employerUrlMatch = email.html.match(/>(http?:.*)<\/p>/)
```

Comment:
Unclear RegEx. Would be nice to get a named regex showing clear intent. Seems like you are matching *any* `<p>` tag starting with `http`
```suggestion
    const pUrlRegex = />(https?:.*)<\/p>/
    const employerUrlMatch = email.html.match(pUrlRegex)
```